src/domains/screens.js

```diff
@@ -0,0 +1,13 @@
+const prisma = require("../utils/prisma");
+
+async function createNewScreen(number) {
+  const screen = await prisma.screen.create({
+    data: {
+      number,
+    },
+  });
+
+  return {screen};
+}
+
+module.exports = { createNewScreen };
```

Comment:
I'm curious as to why you decided to separate these into domains?

Reply:
Separating out into domains means that the controllers have a single responsibility. By extracting the prisma code that is handling database manipulation out into ``domains`` , it allows the code to be more extensible without having large functions in the controllers as more features are added.

Reply:
Ah very clever!